.github/workflows/build-cross-compile.yml

```diff
@@ -131,6 +131,7 @@ jobs:
         id: create-sysroot
         run: >
           sudo debootstrap
+          --no-merged-usr
```

Comment:
I have a question about this. https://wiki.debian.org/UsrMerge says: "This page tracks Debian support for the merged /usr directories scheme, i.e. the /{bin,sbin,lib}/ directories becoming symbolic links to /usr/{bin,sbin,lib}/." 

So turning this off gets us separate `/{bin,sbin,lib}/`, right? We clean up the unnecessary files from a sysroot in the next step, would that step miss `/{bin,sbin}` and subdirs in `/{lib} then?

Reply:
> I have a question about this. https://wiki.debian.org/UsrMerge says: "This page tracks Debian support for the merged /usr directories scheme, i.e. the /{bin,sbin,lib}/ directories becoming symbolic links to /usr/{bin,sbin,lib}/."
> 
> So turning this off gets us separate `/{bin,sbin,lib}/`, right? 

Right. I can confirm that by running two different debootstraps on my Ubuntu 22.04 againt Debian bullseye with and without `--no-merged-usr`. I see independent and non-empty `/{bin,sbin}/` directories. And `/{lib/dev,lib/systemd}/` looks like:
```
$ ls sysroot/lib
cpp  init  lsb  systemd  terminfo  udev  x86_64-linux-gnu

$ tree lib/udev/ lib/systemd/
lib/udev/
└── rules.d
    └── 96-e2scrub.rules
lib/systemd/
└── system
    ├── apt-daily-upgrade.service
    ├── apt-daily-upgrade.timer
    ├── apt-daily.service
    ├── apt-daily.timer
    ├── e2scrub@.service
    ├── e2scrub_all.service
    ├── e2scrub_all.timer
    ├── e2scrub_fail@.service
    ├── e2scrub_reap.service
    ├── fstrim.service
    └── fstrim.timer
```

> We clean up the unnecessary files from a sysroot in the next step, would that step miss `/{bin,sbin}` and subdirs in `/{lib}` then?

Yes, I missed that step. I have pushed another change to fix this. Please take another look. Thanks.